src/fsharp/FSharp.Core.Unittests/FSharp.Core/PrimTypes.fs

```diff
@@ -104,7 +104,7 @@ type LanguagePrimitivesModule() =
         
         // reference type
         let resultRef = LanguagePrimitives.GenericComparison "ABC" "ABCDE"
-        Assert.AreEqual(resultRef,-68)
+        Assert.AreEqual(sign resultRef,-1)
```

Comment:
good catch! i'll fix it, easy
A long time ago I was filling a bug report to NUnit about this, only to find the parameters were reversed :sweat_smile: